src/libs/sfdk/sdk.h

```diff
@@ -48,13 +48,25 @@ class SFDK_EXPORT Sdk : public QObject
         SystemSettingsOnly = 0x2,
         CachedVmInfo = 0x4,
     };
+
+    enum class TextStyle
+    {
+        PrettyText, //NBSP symbol
```

Comment:
So ugly comment following so PrettyText? :) Rather omit that please, proper API doc is planned.